JobConfig/ensemble/SConscript.disabled

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```

Comment:
What is this file for?  Is it a leftover from testing or something you intended to add?